### PR TITLE
Decrease Reverb Decay

### PR DESCRIPTION
**Pull request by ferranlala**
I feel the reverb is not powerful enough. This PR addresses the issue.

### CODE REVIEW

File: lib/reverb/Reverb.cc [hunk 1]
```diff
@@ -438,7 +438,7 @@ void MixxxPlateX2::processBuffer(const sample_t* in, sample_t* out, const uint f
 	// set bandwidth
 	input.bandwidth.set(exp(-M_PI * (1. - (.005 + .994*bandwidthParam))));
 	// set decay
-	sample_t decay = .749*decayParam;
+	sample_t decay = .890*decayParam;
```
Comment (Be-ing) [inline lib/reverb/Reverb.cc]: This function has a lot of magic numbers. Does anyone know where they came from? Guessing and checking until it sounded good? @rryan do you know?

Comment (ferranpujolcamins) [reply lib/reverb/Reverb.cc]: This PR is just changing the max decay paramter until I found a value I liked. Now I find the reverb way more useful: you can generate a long reverb tail and filter it out with the moog filter. Sweet :)

**Comment by Be-ing** [comment]
I tested this and it sounds good to me. Does the decay parameter correspond to a specific value of time in milliseconds? It would be good to note the range in the parameter's tooltip.

**Comment by ferranpujolcamins** [comment]
I have no idea

2018-02-03 18:16 GMT+01:00 Be <notifications@github.com>:

**Comment by Be-ing** [comment]
Okay, it's not terribly important to know that information for this use case.

**Comment by ferranpujolcamins** [comment]
Thanks!

2018-02-03 23:22 GMT+01:00 Be <notifications@github.com>:

**Comment by Be-ing** [comment]
Whoops, I noticed this was targeted at the master branch. I rebased it on 2.1 for the release.

**Pull request merged**